src/app/spartacus/spartacus-configuration.module.ts

```diff
@@ -18,12 +18,29 @@ import { defaultCmsContentProviders, layoutConfig, mediaConfig } from "@spartacu
       currency: ['USD'],
       language: ['en'],
     },
-  }), provideConfig(<I18nConfig>{
+    //<I18nConfig>
+  }), provideConfig({
     i18n: {
       resources: translations,
-      chunks: translationChunksConfig,
-      fallbackLang: 'en'
-    },
+        chunks: translationChunksConfig,
```

Comment:
It would be better if you made two separate commits, one for the guest checkout, and another for the i18n changes.  In this way, if there is a problem with either, they can be dealt with separately.